spec/fixtures/flight_data_return.json

```diff
@@ -0,0 +1,108 @@
+{
+	"data": [{
+
+		"id": 123467,
+		"datetime_departure": "2021-01-07T16:45:10-07:00",
+		"datetime_arrival": "2021-01-07T16:47:12-07:00",
+		"departure_loc": {
+			"city": "Denver, CO",
+			"airport": "DEN"
+		},
+		"arrival_loc": {
+			"city": "Las Vegas, NV",
+			"airport": "LAS"
+		},
+		"airlines": {
+			"to": "frontier",
+			"back": "frontier"
+		},
+		"price": 48,
+		"seat_availability": 7,
+		"duration": "2h 1m",
+		"average_forecast": {
+			"is_available": true,
+			"average_temp": 65,
+			"chance_precip": 20,
+			"sky_coverage": 20,
+			"qualitative_forecast": "sunny"
+		}
+	}, {
+		"id": 12345678,
+		"datetime_departure": "2021-01-07T16:45:10-07:00",
+		"datetime_arrival": "2021-01-07T16:47:12-07:00",
+		"departure_loc": {
+			"city": "Denver, CO",
+			"airport": "DEN"
+		},
+		"arrival_loc": {
+			"city": "Los Angeles, CA",
+			"airport": "LAX"
+		},
+		"airlines": {
+			"to": "frontier",
+			"back": "frontier"
+		},
+		"price": 58,
+		"seat_availability": 7,
+		"duration": "2h 1m",
+		"average_forecast": {
+			"is_available": true,
+			"average_temp": 65,
+			"chance_precip": 20,
+			"sky_coverage": 20,
+			"qualitative_forecast": "sunny"
+		}
+	}, {
+		"id": 2345678,
+		"datetime_departure": "2021-01-07T16:45:10-07:00",
+		"datetime_arrival": "2021-01-07T16:47:12-07:00",
+		"departure_loc": {
+			"city": "Denver, CO",
+			"airport": "DEN"
+		},
+		"arrival_loc": {
+			"city": "Austin, TX",
+			"airport": "AUS"
+		},
+		"airlines": {
+			"to": "frontier",
+			"back": "frontier"
+		},
+		"price": 68,
+		"seat_availability": 7,
+		"duration": "2h 1m",
+		"average_forecast": {
+			"is_available": true,
+			"average_temp": 65,
+			"chance_precip": 20,
+			"sky_coverage": 20,
+			"qualitative_forecast": "sunny"
+		}
+	}, {
+		"id": 3456789,
+		"datetime_departure": "2021-01-07T16:45:10-07:00",
+		"datetime_arrival": "2021-01-07T16:47:12-07:00",
+		"departure_loc": {
+			"city": "Denver, CO",
+			"airport": "DEN"
+		},
+		"arrival_loc": {
+			"city": "Miami, FL",
+			"airport": "MIA"
+		},
+		"airlines": {
+			"to": "frontier",
+			"back": "frontier"
+		},
+		"price": 78,
+		"seat_availability": 7,
+		"duration": "2h 1m",
+		"average_forecast": {
+			"is_available": true,
+			"average_temp": 65,
+			"chance_precip": 20,
+			"sky_coverage": 20,
+			"qualitative_forecast": "sunny"
+		}
+	}]
```

Comment:
Maybe I'm looking at a different version, but this fixture doesn't look like the one that we had talked about? 
Am I just late to the game?

Reply:
I did see in the next PR, this is changed... so totally disregard the last comment